apps/console/src/middleware.ts

```diff
@@ -27,21 +27,22 @@ export default auth(async (req) => {
   const isTfaEnabled = session?.user.isTfaEnabled
 
   if (isLoggedIn) {
-    if (isTfaEnabled) {
+    if (isTfaEnabled && (path === '/tfa' || path === '/login')) {
+      return NextResponse.next()
+    } else if (isTfaEnabled) {
       return NextResponse.redirect(new URL('/tfa', req.url))
     }
```

Comment:
The double `isTfaEnabled` is hard to read, what about: 
 
```suggestion
    if (isTfaEnabled) {
        if (path === '/tfa' || path === '/login')) {
           return NextResponse.next()
       } 
      return NextResponse.redirect(new URL('/tfa', req.url))
   }
```